_data/simple-icons.json

```diff
@@ -2285,6 +2285,11 @@
             "hex": "F94877",
             "source": "https://foursquare.com/about/logos"
         },
+        {
+            "title": "Fozzy",
+            "hex": "F15B29",
+            "source": "https://fozzy.com/"
```

Comment:
Thanks a lot for your PR! So far it looks good to me.

A minor change: I think it would be better to link to https://fozzy.com/eu/partners.shtml?tab=materials as source instead of to the main page, because the SVG version of the logo can be found there more easily than to extract it from the header.